lib/Generate.js

```diff
@@ -3,7 +3,7 @@
 const { supportsColor } = require('chalk');
 const fs = require('fs');
 const path = require('path');
-// eslint-disable-next-line no-unused-vars
+// eslint-disable-next-line no-unused-vars -- imported for the type only
```

Comment:
What I’ve done in other files is to add `void Thing` below (search for `void Project`). `void DependencyProvider` in this case.